azure-pipelines.yml

```diff
@@ -2,14 +2,24 @@
 
 # execute on all branches
```

Comment:
Could you start this comment with Uppercase letter i.e. `execute` so it's consistent with other comments?